src/lib/components/consignments/components/__tests__/boolean_button-tests.tsx

```diff
@@ -0,0 +1,9 @@
+import "jest-snapshots-svg"
+import * as React from "react"
+import * as renderer from "react-test-renderer"
+import BooleanButton from "../boolean_button"
+
+it("looks good as an svg", () => {
```

Comment:
kind of an odd test description? it can still be totally wrong but still look good!